testing/trino-tests/src/test/java/io/trino/security/TestAccessControl.java

```diff
@@ -94,14 +114,27 @@ protected QueryRunner createQueryRunner()
                 .build();
         DistributedQueryRunner queryRunner = DistributedQueryRunner.builder(session)
                 .setNodeCount(1)
+                .setSystemAccessControl(new ForwardingSystemAccessControl() {
+                    @Override
+                    protected SystemAccessControl delegate()
+                    {
+                        return systemAccessControl.get();
+                    }
+                })
                 .build();
         queryRunner.installPlugin(new BlackHolePlugin());
-        queryRunner.createCatalog("blackhole", "blackhole");
+        queryRunner.createCatalog("blackhole", "blackhole", Map.of());
```

Comment:
why the change?

Reply:
I was thinking to remove this plugin from this test. However I failed and I failed to revert changes, as you can see,